io/net/virt-net/Network_virtualization_client_initialized_failover.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2018 IBM
+# Author: Harsha Thyagaraja <harshkid@linux.vnet.ibm.com>
+
+import time
+import netifaces
+from avocado import Test
+from avocado import main
+from avocado.utils import process
+from avocado.utils.software_manager import SoftwareManager
+from avocado.utils.process import CmdError
+
+
+class NetworkVirtualizationFailoverTest(Test):
+
+    """
+    This test performs failover for a Network virtualized device
+    """
+
+    def setUp(self):
+        """
```

Comment:
setUp() is almost same as #828
Can you make them into a single python script with 2 tests ?

Reply:
I am reluctant on including both of them in a single script. 
I appreciate the thought of adding both tests as a single script but I strongly recommend not to judge it that way.  
Failover functionality might change as it is still in Tech Preview. For now, this is how the functionality is coded I suggest we take both those PR requests separately 

i would keep the tests separate with different scripts based on the functionality. :)

Reply:
i also think so it will be good if both test can be part of single script and maintenance  will be easy what i feel @hathyaga i think if Failover functionality change any how we need to change test script accordingly so if it as just cover in one test definition and adoption on new changes will be easy , if you put in single script on in two test definition it not going to matter as avocado run and result publication  prospective any you are owner of test , if still feel it should be in two script should be fine with me :) , those are just suggestion

Reply:
Sure @PraveenPenguin and @narasimhan-v I'll include this test in the already existing network_virtualization_dlpar.py script. 

Thanks for the comments.